lib/solr_wrapper.rb

```diff
@@ -10,6 +10,10 @@ def self.default_instance_options
     @default_instance_options ||= {}
   end
 
+  def self.default_instance_options=(options)
```

Comment:
@jcoyne & @cbeer I can not find the correct syntax to create a class attr_writer  according to the documentation the attr* methods are for instance variables.  I would be happy to change it if someone has the correct syntax.